Exercise02/app/src/main/java/com/example/GameOfThrones/Data.java

```diff
@@ -0,0 +1,11 @@
+package com.example.GameOfThrones;
+
+public class Data {
```

Comment:
Try to think about a more intuitive name for your class.
Data is too general, is it a Character maybe?